base/pkg/entry.jl

```diff
@@ -376,7 +376,11 @@ function update(branch::AbstractString)
                         LibGit2.fetch(repo)
                         LibGit2.merge!(repo, fastforward=true)
                     catch err
-                        show(err)
+                        if isa(err, LibGit2.GitError)
+                            print("Package $pkg cannot be updated. Error: $(err.msg) Class: $(err.class)")
+                        else
+                            show(err)
```

Comment:
>Allow through specific things we know are okay, not everything otherwise you end up having hidden typos again

For what it's worth, while that should be true of library code, I disagree in this situation. This is user interface code, and it should not rethrow, notwithstanding #13991. While that makes debugging difficult, the alternative, we saw in the early days of Pkg1, was  having to delete the entire .julia folder everytime something went wrong.